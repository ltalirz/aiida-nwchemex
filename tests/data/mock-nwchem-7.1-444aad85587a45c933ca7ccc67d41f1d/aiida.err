[0] ARMCI Warning: Freed 1 leaked allocations
