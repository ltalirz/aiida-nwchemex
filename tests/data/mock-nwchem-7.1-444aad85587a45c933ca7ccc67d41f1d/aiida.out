 argument  1 = aiida.in



============================== echo of input deck ==============================
echo
start aiida_calc
title "AiiDA NWChem calculation"
memory 2000.0 mb
geometry units angstroms noautoz noautosym
  system crystal
    lat_a 4.0
    lat_b 4.0
    lat_c 4.0
    alpha 90.0
    beta  90.0
    gamma 90.0
  end
  O 0.0 0.0 0.0
  H 0.0 0.3576070225 -0.276788165
  H 0.0 -0.3576070225 -0.276788165
end
basis
  H library sto-3g
  O library sto-3g
end
task dft
================================================================================


                                         
                                         


             Northwest Computational Chemistry Package (NWChem) 7.0.0
             --------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2019
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = workstation-a-ctalirz
    program         = /usr/bin/nwchem
    date            = Mon Jan 30 15:47:56 2023

    compiled        = Mon_Mar_09_19:57:00_2020
    source          = /build/nwchem-24Ysn1/nwchem-7.0.0
    nwchem branch   = 7.0.0
    nwchem revision = N/A
    ga revision     = 5.7.1
    use scalapack   = T
    input           = aiida.in
    prefix          = aiida_calc.
    data base       = ./aiida_calc.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =   65535996 doubles =    500.0 Mbytes
    stack    =   65536001 doubles =    500.0 Mbytes
    global   =  131072000 doubles =   1000.0 Mbytes (distinct from heap & stack)
    total    =  262143997 doubles =   2000.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


                             AiiDA NWChem calculation
                             ------------------------

  !!!!!!!!!  geom_3d NEEDS TESTING !!!!!!!!!! 


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 O                    8.0000     0.00000000     0.00000000     0.00000000
    2 H                    1.0000     0.00000000     1.43042809     2.89284734
    3 H                    1.0000     0.00000000     2.56957191     2.89284734

      Lattice Parameters 
      ------------------ 

      lattice vectors in angstroms (scale by  1.889725989 to convert to a.u.)

      a1=<   4.000   0.000   0.000 >
      a2=<   0.000   4.000   0.000 >
      a3=<   0.000   0.000   4.000 >
      a=       4.000 b=      4.000 c=       4.000
      alpha=  90.000 beta=  90.000 gamma=  90.000
      omega=    64.0

      reciprocal lattice vectors in a.u.

      b1=<   0.831   0.000  -0.000 >
      b2=<  -0.000   0.831  -0.000 >
      b3=<   0.000   0.000   0.831 >

      Atomic Mass 
      ----------- 

      O                 15.994910
      H                  1.007825



            XYZ format geometry
            -------------------
     3
 geometry
 O                     0.00000000     0.00000000     0.00000000
 H                     0.00000000     1.43042809     2.89284734
 H                     0.00000000     2.56957191     2.89284734

                      Basis "ao basis" -> "" (cartesian)
                      -----
  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  3.42525091E+00  0.154329
  1 S  6.23913730E-01  0.535328
  1 S  1.68855400E-01  0.444635

  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.30709320E+02  0.154329
  1 S  2.38088610E+01  0.535328
  1 S  6.44360830E+00  0.444635

  2 S  5.03315130E+00 -0.099967
  2 S  1.16959610E+00  0.399513
  2 S  3.80389000E-01  0.700115

  3 P  5.03315130E+00  0.155916
  3 P  1.16959610E+00  0.607684
  3 P  3.80389000E-01  0.391957



 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 H                           sto-3g                  1        1   1s
 O                           sto-3g                  3        5   2s1p



                                 NWChem DFT Module
                                 -----------------


                             AiiDA NWChem calculation




 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 H                           sto-3g                  1        1   1s
 O                           sto-3g                  3        5   2s1p


  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     3
          No. of electrons :    10
           Alpha electrons :     5
            Beta electrons :     5
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations:  30
          AO basis - number of functions:     7
                     number of shells:     5
          Convergence on energy requested:  1.00D-06
          Convergence on density requested:  1.00D-05
          Convergence on gradient requested:  5.00D-04

              XC Information
              --------------
                        Slater Exchange Functional  1.000 local    
                      VWN V Correlation Functional  1.000 local    

             Grid Information
             ----------------
          Grid used for XC integration:  medium    
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          O                   0.60       49          10.0       434
          H                   0.35       45           8.0       434
          Grid pruning is: on 
          Number of quadrature shells:   139
          Spatial weights used:  Erf1

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL):  1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho:  1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul:  1.00D-08


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:         -74.71095592

      Non-variational initial energy
      ------------------------------

 Total energy =     -73.846156
 1-e energy   =    -109.536925
 2-e energy   =      32.820317
 HOMO         =      -0.193699
 LUMO         =      -0.193596

   Time after variat. SCF:      0.0
   Time prior to 1st pass:      0.0

 #quartets = 1.180D+02 #integrals = 2.580D+02 #direct =  0.0% #cached =100.0%


 Integral file          = ./aiida_calc.aoints.0
 Record size in doubles =    65536    No. of integs per rec  =    43688
 Max. records in memory =        2    Max. records in file   =  1666364
 No. of bits per label  =        8    No. of bits per value  =       64


 Grid_pts file          = ./aiida_calc.gridpts.0
 Record size in doubles =  12289        No. of grid_pts per rec  =   3070
 Max. records in memory =     23        Max. recs in file   =   8886555


           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       65.12            65121452
          Stack Space remaining (MW):       65.54            65535812

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1    -74.3809064953 -7.73D+01  4.04D-01  1.80D-05     0.1
 d= 0,ls=0.5,diis     2    -74.3808059438  1.01D-04  4.96D-04  8.96D-05     0.1
 d= 0,ls=0.5,diis     3    -74.3808185376 -1.26D-05  4.96D-04  4.76D-05     0.1
 d= 0,ls=0.5,diis     4    -74.3808265445 -8.01D-06  8.01D-04  2.55D-05     0.1
 d= 0,ls=0.5,diis     5    -74.3808342190 -7.67D-06  4.36D-04  8.17D-06     0.2
  Resetting Diis
 d= 0,ls=0.5,diis     6    -74.3808366990 -2.48D-06  3.31D-04  3.25D-06     0.2
 d= 0,ls=0.5,diis     7    -74.3808378755 -1.18D-06  2.50D-04  1.23D-06     0.2
 d= 0,ls=0.5,diis     8    -74.3808384120 -5.36D-07  1.47D-04  3.19D-07     0.2
 d= 0,ls=0.5,diis     9    -74.3808385953 -1.83D-07  9.97D-05  1.56D-07     0.2
 d= 0,ls=0.5,diis    10    -74.3808386832 -8.80D-08  6.88D-05  7.36D-08     0.2
 d= 0,ls=0.5,diis    11    -74.3808387251 -4.18D-08  4.75D-05  3.49D-08     0.2
 d= 0,ls=0.5,diis    12    -74.3808387450 -1.99D-08  3.28D-05  1.66D-08     0.2
  Singularity in Pulay matrix. Error and Fock matrices removed. 
  Singularity in Pulay matrix. Error and Fock matrices removed. 
 d= 0,ls=0.5,diis    13    -74.3808387545 -9.48D-09  2.26D-05  7.88D-09     0.3
  Singularity in Pulay matrix. Error and Fock matrices removed. 
  Singularity in Pulay matrix. Error and Fock matrices removed. 
  Singularity in Pulay matrix. Error and Fock matrices removed. 
 d= 0,ls=0.5,diis    14    -74.3808387590 -4.52D-09  1.56D-05  3.75D-09     0.3
 d= 0,ls=0.5,diis    15    -74.3808387611 -2.15D-09  1.08D-05  1.78D-09     0.3
  Singularity in Pulay matrix. Error and Fock matrices removed. 
 d= 0,ls=0.5,diis    16    -74.3808387622 -1.02D-09  7.45D-06  8.49D-10     0.3


         Total DFT energy =      -74.380838762646
      One electron energy =     -109.783356535073
           Coulomb energy =       41.168756781446
    Exchange-Corr. energy =       -8.636691793331
 Nuclear repulsion energy =        2.870452784312

 Numeric. integr. density =       10.000003656623

     Total iterative time =      0.3s



                       DFT Final Molecular Orbital Analysis
                       ------------------------------------

 Vector    1  Occ=2.000000D+00  E=-1.831628D+01
              MO Center= -3.9D-12, -6.2D-08,  1.1D-07, r^2= 1.5D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.993581  1 O  s                  2      0.025790  1 O  s          

 Vector    2  Occ=2.000000D+00  E=-6.951101D-01
              MO Center=  2.1D-09, -1.9D-04,  2.6D-04, r^2= 4.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      1.028902  1 O  s                  1     -0.268602  1 O  s          

 Vector    3  Occ=2.000000D+00  E=-2.499977D-01
              MO Center=  5.3D-11,  2.0D+00,  2.9D+00, r^2= 8.4D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.593271  2 H  s                  7      0.593882  3 H  s          

 Vector    4  Occ=2.000000D+00  E=-7.476322D-02
              MO Center=  1.5D-09,  1.4D-03,  1.3D-03, r^2= 4.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.990525  1 O  pz                 4     -0.135416  1 O  py         

 Vector    5  Occ=2.000000D+00  E=-7.458973D-02
              MO Center=  2.0D-09,  8.1D-04,  7.0D-04, r^2= 4.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.990721  1 O  py                 5      0.135104  1 O  pz         

 Vector    6  Occ=0.000000D+00  E=-1.650894D-01
              MO Center= -5.6D-09,  5.9D-14, -3.3D-14, r^2= 4.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      1.000000  1 O  px         

 Vector    7  Occ=0.000000D+00  E= 1.495392D-01
              MO Center= -1.3D-11,  2.0D+00,  2.9D+00, r^2= 9.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.927561  2 H  s                  7     -0.926711  3 H  s          


 center of mass
 --------------
 x =   0.00000000 y =   0.42297699 z =   0.61180392

 moments of inertia (a.u.)
 ------------------
          81.400568950877          -0.000000000000          -0.000000000000
          -0.000000000000          53.495642808757         -36.984767269992
          -0.000000000000         -36.984767269992          27.904926142119

     Multipole analysis of the density
     ---------------------------------

     L   x y z        total         alpha         beta         nuclear
     -   - - -        -----         -----         ----         -------
     0   0 0 0     -0.000000     -5.000000     -5.000000     10.000000

     1   1 0 0     -0.000000     -0.000000     -0.000000      0.000000
     1   0 1 0     -0.005816     -3.782360     -3.782360      7.558904
     1   0 0 1     -0.003821     -5.468599     -5.468599     10.933378

     2   2 0 0     -3.660006     -1.830003     -1.830003      0.000000
     2   1 1 0     -0.000012     -0.000006     -0.000006      0.000000
     2   1 0 1      0.000007      0.000004      0.000004      0.000000
     2   0 2 0     -4.333391    -17.609449    -17.609449     30.885508
     2   0 1 1     -0.031434    -20.676805    -20.676805     41.322176
     2   0 0 2     -4.868252    -32.318812    -32.318812     59.769373


 Parallel integral file used       1 records with       0 large values


 Task  times  cpu:        0.2s     wall:        0.3s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  221      221     5943      397     3632        0        0      260     
number of processes/call 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
bytes total:             4.07e+05 9.57e+04 1.91e+05 0.00e+00 0.00e+00 2.08e+03
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 15288 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        21	        51
	current total bytes		         0	         0
	maximum total bytes		   3316352	  22510184
	maximum total K-bytes		      3317	     22511
	maximum total M-bytes		         4	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                                      AUTHORS
                                      -------
     E. Apra, E. J. Bylaska, N. Govind, K. Kowalski, M. Valiev, W. A. de Jong,
      T. P. Straatsma, H. J. J. van Dam, D. Wang, T. L. Windus, N. P. Bauman,
       A. Panyala, J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec,
    K. Lopata, S. A. Fischer, S. Krishnamoorthy, M. Jacquelin, W. Ma, M. Klemm,
       O. Villa, Y. Chen, V. Anisimov, F. Aquino, S. Hirata, M. T. Hackler,
           Eric Hermes, L. Jensen, J. E. Moore, J. C. Becca, V. Konjkov,
            D. Mejia-Rodriguez, T. Risthaus, M. Malagoli, A. Marenich,
   A. Otero-de-la-Roza, J. Mullin, P. Nichols, R. Peverati, J. Pittner, Y. Zhao,
        P.-D. Fan, A. Fonari, M. J. Williamson, R. J. Harrison, J. R. Rehr,
      M. Dupuis, D. Silverstein, D. M. A. Smith, J. Nieplocha, V. Tipparaju,
      M. Krishnan, B. E. Van Kuiken, A. Vazquez-Mayagoitia, M. Swart, Q. Wu,
   T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros,
     G. I. Fann, H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols,
       K. Tsemekhman, K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski,
       T. Clark, D. Clerc, H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood,
      E. Glendening, M. Gutowski, A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju,
        R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield, X. Long, B. Meng,
      T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann, G. Sandrone,
      M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. T. Wong, Z. Zhang.

 Total times  cpu:        0.2s     wall:        0.3s
